packages/metablocks/src/types/types.ts

```diff
@@ -0,0 +1,90 @@
+import { Program } from "@project-serum/anchor";
+import { Connection, PublicKey } from "@solana/web3.js";
+import { MetaBlocks } from "./meta_blocks";
+
+export interface UniverseArgs {
+  connection: Connection;
+  wallet: any;
+  name: string;
+  description: string;
+  websiteUrl: string;
+}
+
+interface InputArgs {
```

Comment:
`InputArgs`: Can we use a more descriptive name?

Reply:
done